src/components/Carousel.js

```diff
@@ -486,6 +490,7 @@ class Carousel extends Component {
     return dragOffset - currentValue * elementWidthWithOffset + additionalOffset - additionalClonesOffset;
   };
 
+  getIdCarouselItem = index => index % this.getChildren().length + 1;
```

Comment:
Why do we use `this.getChildren().length + 1`?

Reply:
This method isn't needed in the last version. I filed PR #547 to kick it out.